src/Chapter1.hs

```diff
@@ -477,8 +477,8 @@ Implement a function that returns the last digit of a given number.
   results. Or you can try to guess the function name, search for it and check
   whether it works for you!
 -}
--- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Integral a => a -> a
+lastDigit n = mod n 10
```

Comment:
Your implementation is almost correct :ok: 
Unfortunately, it returns negative numbers on negative inputs because of how `mod` works. Sometimes corner cases can be tricky to spot and fix...

Reply:
Yes, it seems I forgot to handle corner cases every time they were not mentioned in the task and not present in tests.